plugins/callback/opentelemetry.py

```diff
@@ -556,11 +556,19 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
 
         self.otel_exporter_otlp_traces_protocol = self.get_option('otel_exporter_otlp_traces_protocol')
 
-    def dump_results(self, result):
+    def dump_results(self, task, result):
         """ dump the results if disable_logs is not enabled """
         if self.disable_logs:
             return ""
-        return self._dump_results(result._result)
+        # ansible.builtin.uri contains the response in the json field
+        save = result._result
```

Comment:
How about:
```suggestion
        save = dict(result._result)
```
so that you modify a copy of the result below, instead of accidentally changing some data structure that might be used somewhere else.

Reply:
Aha, I didn't know it's the pointer what I actually used. Thanks for this!